drivers/storage/digitalocean/tests/terraform/.gitignore

```diff
@@ -0,0 +1 @@
+terraform.tfstate*
```

Comment:
Hi @protochron,

I prefer to add entries to the root `.gitignore`, but as long as the entry is relevant to the current directory, that's fine. I've just managed projects in the past where an entry is relevant to some other directory, and the `.gitignore` is removed and the other files get added accidentally.